tutorials/cloud-functions-github-container-builder/index.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Connect Container Builder to Github Through Cloud Functions
+description: Learn how to create Github Statuses from Container Builder events using Cloud Functions.
+author: cbuchacher
+tags: Cloud Functions, Github, Node.js
+date_published: 2018-05-14
+---
+
+Clemens Buchacher | Xavo Software AG
+
+This tutorial demonstrates how to use [Google Cloud Container Builder][gcb] as
+a continuous integration service for Github Repositories. You will implement a
+[Google Cloud Function][gcf] which listens to build events and updates the
+build status in Github using the [Statuses API][statuses]. The function is
+implemented in [Node.js][node].
+
+[gcb]: https://cloud.google.com/container-builder/docs/
+[gcf]: https://cloud.google.com/functions/docs/
+[statuses]: https://developer.github.com/v3/repos/statuses/
+[node]: https://nodejs.org/en/
+
+## Prerequisites
+
+1.  Create a project in the [Google Cloud Platform Console][console].
+1.  [Enable billing][billing] for your project.
+1.  Install the [Google Cloud SDK][sdk].
+
+[console]: https://console.cloud.google.com/
+[billing]: https://support.google.com/cloud/answer/6293499#enable-billing
+[sdk]: https://cloud.google.com/sdk/
+
+## Set up automated builds
+
+You will use [Google Cloud Container Builder][gcb] and the [Build Triggers][bt]
+feature to upload your website automatically every time you push a new git
+commit to the source repository.
+
+> Note: If you do not have a repository on GitHub, you can fork [this sample
+> repository][sample-repo] for the purposes of this tutorial.
+
+Head over to the Container Registry &rarr; [Build Triggers][triggers]
+section on Google Cloud Platform Console and click “Add trigger”:
+
+![Add build trigger on Container Registry section](add-trigger-button.png)
+
+Then select GitHub as the source of your repository. In the next screen you may
+be asked to authorize access to your GitHub account and repositories. This is
+needed for Google Cloud Source Repositories to mirror and create commit hooks on
```

Comment:
For these sentences: 
In the next screen you may be asked to authorize access to your GitHub account and repositories. This is needed for Google Cloud Source Repositories to mirror and create commit hooks on your GitHub repositories.

Suggested edit:
You may need to authorize access to your GitHub account so that Cloud Source Repositories can mirror and create commit hooks on your GitHub repositories.